set-vars.md

```diff
@@ -0,0 +1,159 @@
+session setting variables.
+
+{{site.data.alerts.callout_danger}}In some cases, client drivers can drop and restart the connection to the server. When this happens, any session configurations made with <code>SET</code> statements are lost. It is therefore more reliable to configure the session in the client's connection string. For examples in different languages, see the <a href="build-an-app-with-cockroachdb.html">Build an App with CockroachDB</a> tutorials.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Required Privileges
+
+No [privileges](privileges.html) are required to modify the session settings.
+
+## Synopsis
+
+General syntax:
+
+~~~
+   SET [SESSION] <variable name> = <values...>
+   SET [SESSION] <variable name> TO <values...>
+   SET [SESSION] TIME ZONE <value>
+~~~
+
+The variable name is case insensitive.
+
+Special syntax cases:
+
+| Syntax | Equivalent to |
+|--------|---------------|
+| `SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL ...` | `SET default_transaction_isolation = ...`
+| `SET TIME ZONE ...` | Special syntax because the variable name contains a space. See [`SET TIME ZONE`](#set-time-zone) below.
+
+Note: the `SET` statement for session settings in unrelated to the
+other [`SET TRANSACTION`](set-transaction.html) statement.
+
+## Supported variables
+
+|Variable | Can be viewed with [`SHOW`](show-vars.html)? | Description | Initial value |
+|---------|-----------------------------|-------------|---------------|
+| `database` | Yes | The default database for the current session. | Database in connection string, or empty if not specified.
+| `search_path` | Yes | A list of databases or namespaces that will be searched to resolve unqualified table or function names. For more details, see [Name Resolution](sql-name-resolution.html). | "`{pg_catalog}`" (for ORM compatibility).
+| `time zone` | Yes | The default time zone for the current session. See [`SET TIME ZONE` notes](#set-time-zone) below. | "`UTC`"
+| `default_transaction_isolation` | Yes | The default transaction isolation level for the current session. See [Transaction parameters](transactions.html#transaction-parameters) and [`SET TRANSACTION`](set-transaction.html) for more details. | Settings in connection string, or "`SERIALIZABLE`" if not specified.
+| `syntax` | Yes | The default SQL syntax for the current session. | "`Traditional`"
+| `client_encoding` | No | Ignored; recognized for compatibility with PostgreSQL clients. Only possible value is "`UTF8`". | N/A
+| `client_min_messages` | No | Ignored; recognized for compatibility with PostgreSQL clients. Only possible value is "`on`". | N/A
+| `standard_conforming_strings` | No | Ignored; recognized for compatibility with PostgreSQL clients. | N/A
+| `application_name` | No | Ignored; recognized for compatibility with PostgreSQL clients. | N/A
+| `extra_float_digits` | No | Ignored; recognized for compatibility with PostgreSQL clients. | N/A
+
+## Examples
+
```

Comment:
Examples should all have H3 (`###`) headers indicating what they're an example of.